Modules/Core/Common/src/itkMultiThreaderBase.cxx

```diff
@@ -212,13 +214,7 @@ MultiThreaderBase::SetGlobalMaximumNumberOfThreads(ThreadIdType val)
 {
   itkInitGlobalsMacro(PimplGlobals);
 
-  m_PimplGlobals->m_GlobalMaximumNumberOfThreads = val;
-
-  // clamp between 1 and ITK_MAX_THREADS
-  m_PimplGlobals->m_GlobalMaximumNumberOfThreads =
-    std::min(m_PimplGlobals->m_GlobalMaximumNumberOfThreads, ThreadIdType{ ITK_MAX_THREADS });
-  m_PimplGlobals->m_GlobalMaximumNumberOfThreads =
-    std::max(m_PimplGlobals->m_GlobalMaximumNumberOfThreads, NumericTraits<ThreadIdType>::OneValue());
+  m_PimplGlobals->m_GlobalMaximumNumberOfThreads = std::clamp<ThreadIdType>(val, 1, ITK_MAX_THREADS);
```

Comment:
I like the fact that `std::clamp` is self-documenting 😃 Allowing us to remove comments line that one:

    // clamp between 1 and ITK_MAX_THREADS